src/main/java/course_project/battleship_game/Main.java

```diff
@@ -0,0 +1,11 @@
+package course_project.battleship_game;
+
+import course_project.battleship_game.controller.GameController;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
Opt: it seems indentation is a little bit shifted 
![image](https://user-images.githubusercontent.com/58872509/134147482-dd5fdfe4-a2cb-4d37-b452-c61f1d2482ae.png)

Reply:
![image](https://user-images.githubusercontent.com/69588345/134147799-58acd35b-82b8-4158-8d1a-828fc39cdf30.png)
it looks like this on my computer

Reply:
Really nice game, well-done! Excellent architecture, model/service separation, easy to read code, pretty interface and a lot of options how to play. Also, you're showing good knowledge of Java Core and Java libraries, as well as following Code Conventions and creating proper application architecture. Special thanks for automatic ship placement :) Testing is also fine, as I understand testing of console application can be tough. 

Good job, approved!